src/include/miopen/solver.hpp

```diff
@@ -3119,6 +3119,7 @@ struct PerformanceConfigAsmImplicitGemmGTCWrwXdlopsNHWC : PerformanceConfigAsmIm
     bool IsValidValue() const;
     bool IsValid(const ConvolutionContext& ctx) const;
     size_t ComputeKernelOccupancy() const;
+    void SetParamsForKSplit(const ConvolutionContext& ctx, const size_t& occupancy);
```

Comment:
```suggestion
    private:
    void SetParamsForKSplit(const ConvolutionContext& ctx, const size_t& occupancy);
```
basically, `ComputeKernelOccupancy()` should be private as well.